xray/audit/jas/iacscanner.go

```diff
@@ -84,5 +84,5 @@ func (iac *IacScanManager) createConfigFile(currentWd string) error {
 }
 
 func (iac *IacScanManager) runAnalyzerManager() error {
-	return iac.scanner.analyzerManager.Exec(iac.scanner.configFileName, iacScanCommand, iac.scanner.serverDetails)
+	return iac.scanner.analyzerManager.Exec(iac.scanner.configFileName, iacScanCommand, iac.scanner.analyzerManager.GetAnalyzerManagerDir(), iac.scanner.serverDetails)
```

Comment:
can use iac.scanner.analyzerManager.GetAnalyzerManagerDir from inside

Reply:
same as explained for applicabilitymanager